src/platform/errors/interactiveCellResultError.ts

```diff
@@ -1,5 +1,11 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+import { NotebookCell } from 'vscode';
+
 // Small helper error to use in our class
-export class InteractiveCellResultError extends Error {}
+export class InteractiveCellResultError extends Error {
+    constructor(public readonly cell: NotebookCell) {
+        super('Cell failed to execute');
```

Comment:
Does this message need to be localized?

Reply:
No need, this is not displayed to the user.